docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
+
+- Provide feedback on proposals or ideas
+- Discuss possible use-cases or feature requests
+- Make suggestions for non-technical contributions, including events, writing, or business models
+- Answer questions about the protocol, services and products.
+
+
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
+
+After discussion, if the community agrees that the change should be implemented, the proposer may also make a Treasury proposal to support the work. Check out the treasury proposals section.
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way. 
+
+In addition you should:
+
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT). If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance. See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
+
+Governance approval for a Treasury spend will also  includes list of the deliverables as outlined in the proposal.
```

Comment:
What do you want to say with this statement? Governance approval is a boolean action. Of course, it can only occur, if all proposed deliverables are completed.

Reply:
Yes, but that is only obvious if you know about the governance. It is a new concept and the readers potentially will not know about this. Therefore, we must be explicit in our speech and not let the reader knows.

Reply:
IMO, Governance mainly refers to Democracy (e.g. the entire community can vote in a public referendum). In the case of a Treasury proposal, voices can be raised but eventually the Council votes on it.

Reply:
In any case, the language is slightly off, isn't it?
```suggestion
Governance approval for a Treasury spend will also include a list of the deliverables as outlined in the proposal.
```

Reply:
We can get rid of this statement.  :)